docs/external_hms_glue.md

```diff
@@ -0,0 +1,48 @@
+# External HMS and Glue Integration
+### TL;DR
```

Comment:
can you link this documentation from the main readme.md?

Reply:
Sure